brownie-config.yml

```diff
@@ -28,7 +30,7 @@ reports:
 contracts:
   # contracts used by the strategy
   registry: "0x3199437193625DCcD6F9C9e98BDf93582200Eb1f"
-  gauge: "0xbF7E49483881C76487b0989CD7d9A8239B20CA41"
+  gauge: "0xCE5F24B7A95e9cBa7df4B54E911B4A3Dc8CDAf6f"
```

Comment:
gauge migrated to a new address => https://gov.curve.fi/t/sidechain-gauge-upgrade-and-migration/3869